### ISSUE TITLE
bug(MatSelect): With VoiceOver + Safari, VoiceOver reads 2 letters of the label.

### ISSUE THREAD

**Issue by lynnkim-prev** [angular/components#24391]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

With VoiceOver + Safari, VoiceOver reads 2 letters of the label.

### Reproduction

Steps to reproduce:
1. Access [https://material.angular.io/components/select/overview](https://material.angular.io/components/select/overview) in Safari
2. Navigate to Basic mat-select with tab

### Expected Behavior

<img width="1295" alt="chrome" src="https://user-images.githubusercontent.com/43070523/153297148-0ed4bd4a-9fdd-44f2-8413-f6f20f720640.png">

### Actual Behavior

VO reads 
Steak
**Fa**, Favorite food Steak, ... 
<img width="1295" alt="safari" src="https://user-images.githubusercontent.com/43070523/153296979-7d132825-50ea-4efe-94e2-7e10fe4e30ff.png">

### Environment

- Angular: v11, v12, v13 (observed from the latest 3 versions on the official website)
- CDK/Material: Material
- Browser(s): Safari
- Operating System (e.g. Windows, macOS, Ubuntu): macOS

**Comment by crisbeto** [comment]
I can reproduce it too, although nothing specific stands out that could be causing it so it may be a browser bug. All we're doing is setting an `aria-labelledby` pointing to the form field label.

**Comment by omarboulbaze** [comment]
It seems that the VoiceOver with Chrome reads out the value twice : Favourite Food Steak **Steak** on Chrome.
Source: https://material.angular.io/components/select/examples
![image](https://user-images.githubusercontent.com/25458706/194570909-746f7003-2cc3-4977-80eb-b63cb28115a6.png)

**Comment by zarend** [comment]
Hello,

I cannot reproduce. We might have already fixed this. @lynnkim-prev does the issue still happen for you?

- macos 12.6.1 (21G217)
 - Safari Version 16.1 (17614.2.9.1.13, 17614)
 - VoiceOver
 
-Zach

**Comment by zarend** [comment]
Hello @lynnkim-prev ,

Thank you for reporting this issue. Unfortunately we are not able to take action if we cannot reproduce it. This might have already been fixed. We hope Angular Material meets your needs. If you continue to have problems please reopen this issue with reproduction steps or file a new issue.

- Zach
